src/github.com/couchbase/sync_gateway/db/index_changes.go

```diff
@@ -11,12 +11,15 @@ package db
 
 import (
 	"fmt"
-	"time"
 
 	"github.com/couchbase/sync_gateway/base"
 	"github.com/couchbase/sync_gateway/channels"
 )
 
+const (
+	kContinuousHashFrequency = 100 // For a continuous feed, how frequently a hash value is calculated
```

Comment:
can you add the units to the variable name?  eg, kContinuousHashFrequencyNs  // ... in nanoseconds